contracts/test/bridge-test.ts

```diff
@@ -0,0 +1,312 @@
+import { expect } from "chai";
```

Comment:
typescript tests ? argh. lol :)

Reply:
Helps my vs code navigate easier, which is handy when I have no clue what I'm doing 😆